.github/workflows/ci.yml

```diff
@@ -61,7 +61,7 @@ jobs:
     - name: Install development dependencies
       run: |
         sudo apt-get update
-        sudo apt-get install -y --no-install-recommends libxmu-dev libxi-dev libgl-dev libglu1-mesa-dev libgles2-mesa-dev libwayland-dev libxkbcommon-dev libegl1-mesa-dev libosmesa6 mesa-utils libglvnd-dev x11-utils catch2
```

Comment:
New dependencies should be added to the documentation. Also, are you sure no other linux workflows need it?

Reply:
> New dependencies should be added to the documentation.

There are some other packages here that aren't listed in the build readme. Also it already says that _Additional dependencies will likely need to be installed to build the libraries from source as well._ in the Linux section about vcpkg.

> Also, are you sure no other linux workflows need it?

No other check is complaining, so I guess it's fine.

Reply:
What's the new library do and why is it needed?

Reply:
Am I correct in assuming that this is due to the pipewire/alsa/openal dependencies requiring libtool in the new version? Do you know in what cases that's required? It could just be included in some of the ubuntu images by default so they didn't fail the workflow, but that doesn't mean they aren't being used.

Reply:
If I'm reading the logs correctly, it's needed to build libxcrypt, which is not a dependency of OpenAL: https://vcpkg.io/en/package/openal-soft.html.

Reply:
And what is libxcrypt a dependency of? We don't use it directly.

Reply:
Alright, traced it: sdl2 -> dbus -> libsystemd -> libxcrypt.

Reply:
If it's building against a newer version of SDL2 (and presumably all the other libraries), is that going to cause problems for people with older versions installed?

Reply:
Idk, I'm not able to test every platform and configuration, but even if it's going to cause any problems, a simple removal of the `build/` dir (or even re-running `cmake --profile`) should fix them.